src/routes/profile/link.svelte

```diff
@@ -0,0 +1,41 @@
+<script lang="ts">
+  import { Content } from 'carbon-components-svelte'
+  import { session } from '$app/stores'
+  import LoginButton from '$lib/LoginButton.svelte'
+  import { onMount } from 'svelte'
```

Comment:
minor nit, I would lean towards importing in alphabetical order from npm packages -> aliased packages -> relative packages -> type imports